src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -41,8 +40,6 @@ class EnrollmentHistory extends React.Component {
     return (
       <div>
         <h3 className="section-header">Enrollment History</h3>
-        {/* Find out if this line should be present if the blue box with same info is rendered */}
```

Comment:
@phiden said that we can include the box on every box, so I don't think this is needed anymore.